tcl/main.tcl

```diff
@@ -1135,6 +1135,7 @@ proc CreateMainBoard { {w} } {
   CreateGameInfo
 
   ::board::new $w.board $::boardSize
+  ::board::showMarks $w.board 1
```

Comment:
Just to be clear: I have not tried the code: maybe instead of "1" this should be also gameInfo(showMarks) ?

Reply:
I see, will test it